custom/icds/location_reassignment/utils.py

```diff
@@ -52,21 +52,26 @@ def get_household_and_child_case_ids_by_owner(domain, household_case_id, owner_i
 
 
 def get_household_child_cases_by_owner(domain, household_case_id, owner_id, case_types):
-    def get_child_cases(ids):
+    def get_child_cases(ids, exclude_ids):
         return [case for case in
                 CaseAccessorSQL.get_reverse_indexed_cases(domain, ids)
-                if case.owner_id == owner_id]
+                if case.case_id not in exclude_ids and case.owner_id == owner_id]
 
+    # keep a list of case ids already iterated to avoid duplicates
```

Comment:
Given that `cases` is already a set, I don't quite follow why this is needed, won't duplicates already be eliminated?